src/BinSkim.Rules/RuleResources.resx

```diff
@@ -266,43 +310,63 @@ Modules triggering this check were:
   </data>
   <data name="BA2016_Pass" xml:space="preserve">
     <value>'{0}' is marked as NX compatible, helping to prevent attackers from executing code that is injected into data segments.</value>
   </data>
   <data name="BA2018_EnableSafeSEH_Description" xml:space="preserve">
-    <value>X86 binaries should enable the SafeSEH mitigation to minimize exploitable memory corruption issues. SafeSEH makes it more difficult to exploit vulnerabilities that permit overwriting SEH control blocks on the stack, by verifying that the location to which a thrown SEH exception would jump is indeed defined as an exception handler in the source program (and not shellcode). To resolve this issue, supply the /SafeSEH flag on the linker command line. Note that you will need to configure your build system to supply this flag for x86 builds only, as the /SafeSEH flag is invalid when linking for ARM and x64.</value>
+    <value>X86 binaries should enable the SafeSEH mitigation to minimize exploitable memory corruption issues. SafeSEH makes it more difficult to exploit vulnerabilities that permit overwriting SEH control blocks on the stack, by verifying that the location to which a thrown SEH exception would jump is indeed defined as an exception handler in the source program (and not shellcode).
+To resolve this issue, supply the /SafeSEH flag on link.exe command line.
+For VC projects use ItemDefinitionGroup - Link = ImageHasSafeExceptionHandlers property with 'true' value.
+Note that you will need to configure your build system to supply this flag for x86 builds only, as the /SafeSEH flag is invalid when linking for ARM and x64.</value>
   </data>
   <data name="BA2018_Error" xml:space="preserve">
-    <value>'{0}' is an x86 binary which {1}, indicating that it does not enable the SafeSEH mitigation. SafeSEH makes it more difficult to exploit memory corruption vulnerabilities that can overwrite SEH control blocks on the stack, by verifying that the location to which a thrown SEH exception would jump is indeed defined as an exception handler in the source program (and not shellcode). To resolve this issue, supply the /SafeSEH flag on the linker command line. Note that you will need to configure your build system to supply this flag for x86 builds only, as the /SafeSEH flag is invalid when linking for ARM and x64.</value>
+    <value>'{0}' is an x86 binary which {1}, indicating that it does not enable the SafeSEH mitigation. SafeSEH makes it more difficult to exploit memory corruption vulnerabilities that can overwrite SEH control blocks on the stack, by verifying that the location to which a thrown SEH exception would jump is indeed defined as an exception handler in the source program (and not shellcode).
+To resolve this issue, supply the /SafeSEH flag on link.exe command line.
+For VC projects use ItemDefinitionGroup - Link = ImageHasSafeExceptionHandlers property with 'true' value.
```

Comment:
>VC projects [](http://example.com/codeflow?start=4&length=11)

For user that is not using Microsoft toolchain. Do we think 
`VC projects` is clear for them or we want a longer name more clear,
I am not sure, an open question.

Reply:
What would you suggest? "VC" is short name for "Visual Studio C++" and if you search for "VC Project" you get
https://learn.microsoft.com/en-us/cpp/build/creating-and-managing-visual-cpp-projects?view=msvc-170. 
Alternatively, I can say "for vcxproj" - would it be better?

Reply:
Just tested "VC Project" with non-Microsoft search engine with Incognito, the fist 3 results all are something else other than the topic. The Microsoft search engine did return all related topic. 
But "VC Project" is clear as well.
+ @suvamM if any preference.

Reply:
How about I leave it as "VC project" for now? As soon as people can quickly tell if this info is applicable to them or not, we are good here. And we can always search/replace it with something else if we get feedback.

Reply:
sounds good to me